tensorflow/tools/ci_build/linux/rocm/run_cc_core.sh

```diff
@@ -19,21 +19,27 @@ set -e
 set -x
 
 N_JOBS=$(grep -c ^processor /proc/cpuinfo)
+N_GPUS=$(lspci|grep 'VGA'|grep 'AMD/ATI'|wc -l)
 
 echo ""
 echo "Bazel will use ${N_JOBS} concurrent job(s)."
```

Comment:
Did you want to remove this line?

Reply:
yes....thank you for catching it.

let me fix that and resubmit.

Reply:
done